vertx-auth-oauth2/src/main/java/io/vertx/ext/auth/oauth2/DCRResponse.java

```diff
@@ -0,0 +1,50 @@
+package io.vertx.ext.auth.oauth2;
+
+import io.vertx.codegen.annotations.DataObject;
+import io.vertx.codegen.json.annotations.JsonGen;
+import io.vertx.core.json.JsonObject;
+
+@DataObject
+@JsonGen(publicConverter = false)
+public class DCRResponse {
+  private String id;
+  private String clientId;
+  private Boolean enabled;
```

Comment:
if enabled is not a tri-state `true|false|null` I'd prefer a primitive `boolean`. This reduces null checks and possible NPE bugs